app/components/appMainModule/committeePageModule/committeePage.html

```diff
@@ -1,5 +1,5 @@
 
-<div class="fluid-container">
+<div class="container">
```

Comment:
On my larger screen the mocks are in a narrow container, but turns out that's just invision adding it's own container and being confusing. @elinaru can you clarify what the container widths should be? Fixed containers in bootstrap are up to `1170px` on the largest desktop screens. The other option would be to have a fluid/full-width container.